src/yuzu/util/limitable_input_dialog.cpp

```diff
@@ -37,18 +39,36 @@ void LimitableInputDialog::ConnectEvents() {
 }
 
 QString LimitableInputDialog::GetText(QWidget* parent, const QString& title, const QString& text,
-                                      int min_character_limit, int max_character_limit) {
+                                      int min_character_limit, int max_character_limit,
+                                      InputLimiter limit_type) {
     Q_ASSERT(min_character_limit <= max_character_limit);
 
     LimitableInputDialog dialog{parent};
     dialog.setWindowTitle(title);
     dialog.text_label->setText(text);
     dialog.text_entry->setMaxLength(max_character_limit);
+    dialog.text_label_invalid->show();
+
+    switch (limit_type) {
+    case InputLimiter::Filesystem:
+        dialog.invalid_characters = QStringLiteral("<>:;\"/\\|,.!?*");
+        break;
+    default:
+        dialog.invalid_characters.clear();
+        dialog.text_label_invalid->hide();
+        break;
+    }
+    dialog.text_label_invalid->setText(
+        QStringLiteral("The text can't contain any of the following characters:\n") +
```

Comment:
```suggestion
        tr("The text can't contain any of the following characters:\n%1").arg(dialog.invalid_characters);
```

Allows for localization of the limitation text.